Calibration/HcalAlCaRecoProducers/plugins/AlCaHcalIsotrkProducer.cc

```diff
@@ -654,7 +654,7 @@ void AlCaHcalIsotrkProducer::produce(edm::Event& iEvent, edm::EventSetup const&
             //loop over all trigger filters in event (i.e. filters passed)
             for (unsigned int ifilter = 0; ifilter < triggerEvent.sizeFilters(); ++ifilter) {
               std::vector<int> Keys;
-              std::string label = triggerEvent.filterTag(ifilter).label();
+              auto const label = triggerEvent.filterLabel(ifilter);
               //loop over keys to objects passing this filter
               for (unsigned int imodule = 0; imodule < moduleLabels.size(); imodule++) {
                 if (label.find(moduleLabels[imodule]) != std::string::npos) {
```

Comment:
Well, Chris's comment would also apply here as ```label`` is now a ```string_view``` - no longer a ```string``` with ```std::string::npos```... and anywhwere the ```findLabel``` is now used like this. ?

Reply:
Thanks, I fixed this in 6b7396d344b. I didn't find other instances where `filterLabel(idx)` is used which would require further changes.